README.rst

```diff
@@ -31,6 +31,15 @@ Optional:
 -  SPIRV-Cross >= 2019-03-22 (for OpenGL ES backends)
 -  libshaderc (for OpenGL ES backends)
 
+Platform-Specific Instructions
```

Comment:
Could the readme changes please go in a separate PR? I am very much in favor of documenting the dependency subprojects, though.